src/main/resources/application.properties

```diff
@@ -1,2 +1,10 @@
 com.zilen.appid = 50b4a9bfff0df818b269d4a468128442
 com.zilen.url = https://api.openweathermap.org/data/2.5/weather?q=
+
+spring.datasource.url=jdbc:postgresql://localhost:5432/weather
+spring.datasource.username=zilen
+spring.datasource.password=10203
```

Comment:
It is bad practice to take passwords as plain text. Consider encryption at least Base64.